front/components/assistant_builder/InstructionScreen.tsx

```diff
@@ -448,27 +449,27 @@ function Suggestions({
           {(() => {
             if (error) {
               return (
-                <ContentMessage size="sm" title="Error" variant="red">
-                  Error loading new suggestions:
-                  {error.message}
-                </ContentMessage>
+                <AnimatedSuggestion
+                  variant="red"
+                  suggestion={`Error loading new suggestions:\n${error.message}`}
+                />
               );
             }
             if (suggestionsStatus === "instructions_are_good") {
               return (
-                <ContentMessage size="sm" variant="slate" title="">
-                  Looking good! 🎉
-                </ContentMessage>
+                <AnimatedSuggestion
+                  variant="slate"
+                  suggestion="Looking good! 🎉"
+                />
               );
             }
             return (
-              <div className="flex w-max gap-3">
-                {suggestions.map((suggestion, index) => (
-                  <div key={index} className="w-[320px] transition-all">
-                    <ContentMessage size="sm" title="" variant="sky">
-                      {suggestion}
-                    </ContentMessage>
-                  </div>
+              <div className="flex w-max">
+                {suggestions.map((suggestion) => (
+                  <AnimatedSuggestion
+                    suggestion={suggestion}
+                    key={suggestion}
```

Comment:
Can we use the index here? so we keep to a small size?

Reply:
went with a hash, so same key for same content